shoes-core/lib/shoes/common/ui_element.rb

```diff
@@ -41,8 +41,7 @@ def initialize(app, parent, *args)
       #
       # It is intended for performing any additions to the styles hash before
       # that gets sent on to style_init.
-      def before_initialize(_styles, *_)
-      end
+      def before_initialize(_styles, *_); end
```

Comment:
not personally a fan of this style gotta say, would still be ok with it but imo `;` shouldn't be in production code :)

Reply:
Also personally don't care for this. If we could revert this and potentially move `Style/EmptyMethod` with enabled false over to the main file, that'd be awesome. (If you don't want to make the move, I'm fine with doing that separately too.)